src/content/contributing/release-process/_index.en.md

```diff
@@ -212,11 +212,50 @@ If this is a final release, add a section for it on this website's [release note
 Alternatively you can edit the file and create a pull request directly on GitHub
 [here](https://github.com/submariner-io/submariner-website/edit/master/src/content/releases/_index.en.md)
 
-### Step 8: Verify the release
+### Step 8: Update the Submariner operator on OperatorHub.io
+
+The Git repository called [community-operators](https://github.com/operator-framework/community-operators)
+is the source for community based operators. This repository is split into two sections:
+
+* Operators for deployment to a vanilla Kubernetes environment (upstream-community-operators)
+* Operators for deployment to OpenShift (community-operators)
+
+OperatorHub.io displays content that is targeted for vanilla Kubernetes.
+
+1) Clone the [submariner-operator](https://github.com/submariner-io/submariner-operator) project
+2) Make sure you have operator-sdk installed on your machine
```

Comment:
```suggestion
2) Make sure you have the operator-sdk installed on your machine
```

Can you provide instructions on how to install the operator-sdk and what version?

Reply:
👆

Reply:
Sorry I missed that comment